master/internal/experiment/bulk_action.go

```diff
@@ -404,7 +429,7 @@ func DeleteExperiments(ctx context.Context,
 			validIDs = append(validIDs, check.ID)
 		}
 	}
-	if filters == nil {
+	if filters == nil && searchFilter == nil {
```

Comment:
why is searchfilter being checked here and not in the other bulk action functions?

Reply:
This is the kind of bug I'm concerned we'd run into by having two filter parameters, where we check one, but not the other.